crates/lumina_shared/src/type_registry.rs

```diff
@@ -24,6 +25,8 @@ impl Plugin for TypeRegistryPlugin {
             .register_type::<WeaponType>()
             .register_type::<Spaceship>()
             .register_type::<SpaceshipType>()
+            .register_type::<Boost>()
+            .register_type::<BoostState>()
```

Comment:
Hmm, why is `BoostState` a component?

Reply:
If I didnt register as component, not sure why blenvy will only get '' empty string, but make it component will make it a drop down selection as the default but since BoostState is no needed now, it will be removed.

Reply:
ohh, u could just register type without making it a component haha, ok understood now.